deployment/docker/traffic-replayer/run_app.sh

```diff
@@ -23,7 +24,7 @@ do
   # process with the Replayer jar before ultimately printing the produced triples to stdout via tee and sending to TC via netcat
   # Note: Unbuffer has not been verified to actually be effective here. stdbuf was ruled out for its caveat with tee (https://linux.die.net/man/1/stdbuf)
   unbuffer tail -f triples.log | tee /dev/stderr | nc -v localhost 9220 &
-  nc -v -l -p 9210 | java -jar trafficReplayer-uber.jar "$target_endpoint" -o triples.log --insecure
+  nc -v -l -p 9210 | java -cp build/libs/KafkaPrinter.jar:"$jars" org.opensearch.migrations.replay.TrafficReplayer "$target_endpoint" -o triples.log --insecure
```

Comment:
Shouldn't this be the TrafficReplayer jar instead of KafkaPrinter?

Reply:
Thanks for catching this Tanner. Just updated it, was not included in the commit. Tested it was still working as intended after the update as well.